ircd/listener.c

```diff
@@ -608,7 +609,11 @@ accept_precallback(rb_fde_t *F, struct sockaddr *addr, rb_socklen_t addrlen, voi
 			last_oper_notice = rb_current_time();
 		}
 
-		rb_write(F, "ERROR :All connections in use\r\n", 31);
+		if(listener->ssl)
+			rb_write(F, sslerrcode, sizeof(sslerrcode));
```

Comment:
Alert 80 "internal_error" would be a better choice for this situation (but not any of the others below).